src/main/resources/swagger/api-docs.yaml

```diff
@@ -4115,10 +4115,30 @@ definitions:
       ok:
         type: boolean
         description: whether any system(s) need attention
-
+        default: false
       systems:
         type: object
         description: Map[String, SubsystemStatus]
+        default: {
```

Comment:
The syntax does work - it's what is displayed as an example value. I'm pretty sure there's a yaml way to do this instead of json though ... looking into it.